table-store/src/integration-test/java/com/scalar/dl/tablestore/TableStoreEndToEndTest.java

```diff
@@ -0,0 +1,1093 @@
+            + ACCOUNT_NAME
+            + " STRING,"
+            + ACTIVE
+            + " BOOLEAN)");
+    clientService.executeStatement(
+        "CREATE TABLE "
+            + PAYMENT_TABLE
+            + "("
+            + PAYMENT_ID
+            + " INT PRIMARY KEY,"
+            + SENDER
+            + " INT,"
+            + RECEIVER
+            + " INT,"
+            + DATE
+            + " STRING)");
+  }
+
+  protected void populateAccountRecords() throws IOException {
+    for (int i = 0; i < NUM_ACCOUNTS; i++) {
+      insertAccountRecord(i, INITIAL_BALANCE);
+    }
+  }
+
+  private int getAccountType(int accountId) {
+    return accountId % NUM_TYPES;
+  }
+
+  private String getAccountName(int accountId) {
+    int accountType = getAccountType(accountId);
+    return accountId + "_" + accountType;
+  }
+
+  private boolean isActive(int accountId) {
+    return accountId % 2 == 0;
+  }
+
+  private JsonNode createAccountJson(int accountId, int balance) {
+    return jacksonSerDe
+        .getObjectMapper()
+        .createObjectNode()
+        .put(ACCOUNT_ID, accountId)
+        .put(ACCOUNT_TYPE, getAccountType(accountId))
+        .put(ACCOUNT_NAME, getAccountName(accountId))
+        .put(ACTIVE, isActive(accountId))
+        .put(BALANCE, balance);
+  }
+
+  private void insertAccountRecord(int accountId, int balance) {
+    clientService.executeStatement(
+        "INSERT INTO "
+            + ACCOUNT_TABLE
+            + " VALUES "
+            + toPartiQLJsonString(createAccountJson(accountId, balance)));
+  }
+
+  private void updateAccountRecord(int accountId, int balance) {
+    clientService.executeStatement(
+        "UPDATE "
+            + ACCOUNT_TABLE
+            + " SET "
+            + BALANCE
+            + " = "
+            + balance
+            + " WHERE "
+            + ACCOUNT_ID
+            + " = "
+            + accountId);
+  }
+
+  private ObjectNode createPaymentJson(int paymentId) {
+    return jacksonSerDe.getObjectMapper().createObjectNode().put(PAYMENT_ID, paymentId);
+  }
+
+  private JsonNode createPaymentJson(
+      int paymentId, int sender, int receiver, int amount, String date) {
+    return createPaymentJson(paymentId)
+        .put(SENDER, sender)
+        .put(RECEIVER, receiver)
+        .put(AMOUNT, amount)
+        .put(DATE, date);
+  }
+
+  private void insertPaymentRecord(
+      int paymentId, int sender, int receiver, int amount, String date) {
+    clientService.executeStatement(
+        "INSERT INTO "
+            + PAYMENT_TABLE
+            + " VALUES "
+            + toPartiQLJsonString(createPaymentJson(paymentId, sender, receiver, amount, date)));
+  }
+
+  private void insertPaymentRecordWithNullValues(int paymentId) {
+    clientService.executeStatement(
+        "INSERT INTO "
+            + PAYMENT_TABLE
+            + " VALUES "
+            + toPartiQLJsonString(createPaymentJson(paymentId)));
+  }
+
+  private String toPartiQLJsonString(JsonNode node) {
```

Comment:
Is this going to be not used by [the update ](https://github.com/scalar-labs/scalardl/pull/246)?

Reply:
That's right! It will be removed in the next PR https://github.com/scalar-labs/scalardl/pull/246/files#r2306118774. Sorry, it was a bit confusing.